src/main/webapp/app/shared/tab-bar/tab-bar.scss

```diff
@@ -0,0 +1,55 @@
+.tab-bar {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0.5rem 0;
+    margin-bottom: 0.5rem;
+
+    border-bottom: 1px solid var(--overview-border-color);
+
+    a {
+        display: inline-block;
+        position: relative;
+        color: var(--tab-bar-color);
+        text-decoration: none !important; // override bootstrap
+
+        padding-bottom: 2px;
+        margin: 0.5rem 0.5rem 0.5rem 0;
+
+        &:hover {
+            color: var(--tab-bar-color-hover) !important; // override bootstrap
+            box-shadow: 0 2px 0 var(--tab-bar-color-hover);
+        }
+
+        &.active {
+            color: var(--tab-bar-color-active) !important; // override :hover
+            box-shadow: 0 2px 0 var(--tab-bar-color-active);
+        }
+
+        &.newMessage:after {
```

Comment:
Can you add a comment what this is? I'm a bit confused about the `transform` and that there is no `left` value.

Reply:
Added a comment, also edited the `transform` and added a `right` value for less confusion